README.md

```diff
@@ -0,0 +1,17 @@
+# BoutrosLab.statistics.general
+
+`BoutrosLab.statistics.general` is a collection of statistical utility functions.
+
+## License
+
+Author: Paul C. Boutros (PBoutros@mednet.ucla.edu)
+
+BoutrosLab.statistics.general is licensed under the GNU General Public License version 2. See the file LICENSE.md for the terms of the GNU GPL license.
+
+`BoutrosLab.statistics.general` is a collection of statistical utility functions.
+
+Copyright (C) 2023 University of California Los Angeles ("Boutros Lab") All rights reserved.
```

Comment:
Clarifying question about how copyright should be done on these licenses. 

While I was updating the license of the [BPG README](https://github.com/uclahs-cds/package-BoutrosLab-plotting-general#license), I used a range of years rather than just the current year, as well as include the period that the lab was at OICR.

Is there a preference?